tests/journal.rs

```diff
@@ -55,6 +56,11 @@ fn ts() {
     log!(log::Level::Info, "rust-systemd test_seek entry");
     assert!(j.seek(journal::JournalSeek::Head).is_ok());
     let _s = j.timestamp().unwrap();
+    assert!(j.seek(journal::JournalSeek::Tail).is_ok());
+    let (u, entry_boot_id) = j.monotonic_timestamp().unwrap();
+    assert!(u > 0);
+    let boot_id = id128::Id128::from_boot().unwrap();
+    assert!(boot_id == entry_boot_id);
```

Comment:
Aside: I spent a while trying to get this to pass locally. It turned out to be due to the fact I was running it from a container. For reasons I still don't fully understand, the boot ID from the journal entry was different there even though I had full access to the host journals (and e.g. `journalctl --list-boots` in the container gave the right boot IDs).

I ended up testing this in a VM and it passed fine there!

Reply:
Did you make `/etc/machine-id` the same on the host and container?

Reply:
Yup! That was a requirement to get `journalctl` to show any logs at all in the container.